tcms/rpc/tests/test_testcasestatus.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=attribute-defined-outside-init
+
+from xmlrpc.client import ProtocolError
+from tcms.rpc.tests.utils import APITestCase, APIPermissionsTestCase
+from tcms.testcases.models import TestCaseStatus
+
+
+class TestCaseStatusFilter(APITestCase):
+    """Test TestCaseStatus.filter method"""
+
+    def _fixture_setup(self):
+        super()._fixture_setup()
+        self.total_case_status = TestCaseStatus.objects.all().count()
```

Comment:
You are not using the total_case_status variable anymore and the entire setup method becomes unnecessary.

Reply:
Sorry about that. Fixed it!